cluster/cluster.go

```diff
@@ -85,10 +90,7 @@ func (client *cluster) DeletePod(podName string) error {
 
 func (client *cluster) createNamespace() error {
 	ns, err := client.Clientset.CoreV1().Namespaces().Get(Namespace, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-	if ns != nil {
+	if ns.Name == Namespace {
```

Comment:
It returns err when it's not found, but instead of checking if the error is a not found I think this check is much more specific. Also, `ns` is not nil when it's not found, so this wont crash when the namespace is not found, but its name will be `""`

Reply:
got it, so better write that in code or comment